lib/hammer_cli_katello/activation_key.rb

```diff
@@ -234,6 +234,10 @@ class ContentOverrideCommand < HammerCLIKatello::SingleResourceCommand
       desc _("Override product content defaults")
       command_name "content-override"
 
+      validate_options do
+        all(:option_label, :option_value).required
```

Comment:
if it's not required in the UI, why should it be required here? i'm not sure i follow.